__init__.py

```
@@ -124,38 +127,21 @@ def launch_app(self, app: str) -> bool:
-            LOG.debug(f"Closing window '{win[0]}' : {win[-1]}")
-            self.close_window(win[0], wmctrl=self.wmctrl)
-            if not self.settings.get("terminate_all", False):
-                break
-
-        self.acknowledge()
-        return True
+    def is_running(self, app: str) -> bool:
+        """ check if a application is running"""
+        if self.match_window(app):
+            return True
+        for p in self.match_process(app):
```

Ruff

Loop control variable `p` not used within loop body

Rename unused `p` to `_p`

(B007)